packages/engine-test-utils/src/__tests__/engine-server/markdown/__snapshots__/dendronPub.spec.ts.snap

```diff
@@ -536,7 +536,7 @@ exports[`GIVEN dendronPub (old tests - need to be migrated) note reference fail:
 VFile {
   "contents": "<h1 id=\\"ref\\">Ref</h1>
 <p></p><blockquote>
-No note with name bar found in vault vault2
+No note with name bar found in cache during parsing.
```

Comment:
for a separate pr - can we have a more detailed error message here? (include vault name)